index.html

```diff
@@ -28,71 +29,75 @@
     </p>
 
     <p><b>Pin Brands:</b></p>
-    <ul>
-      <li>
-        <a href="db.html">D+B</a>
-        (001~011)
+    <ul class="nav nav-tabs" id="pintabs" role="tablist">
+      <li class="nav-item">
+        <a class="nav-link" data-toggle="tab" href="#db">D+B (001~011)</a>
       </li>
-      <li>
-        <a href="np.html">Natural Puppy</a>
-        (012~034)
+      <li class="nav-item">
+        <a class="nav-link" data-toggle="tab" href="#"
+          >Natural Puppy (012~034)</a
```

Comment:
Hey, can we format this so that the tag isn't broken up? i.e. 
```html
<a class="nav-link" data-toggle="tab" href="#">
  Natural Puppy (012~034)
</a>
```